docs/section11/Rule_11-1.md

```diff
@@ -0,0 +1,63 @@
+  - set a list user_proposed_comparison to [].  If the end of the rule is reached and this list (and the corresponding baseline list) still have zero length, then the rule passes.  Otherwise the list will include a series of messages for the user about why it didn't pass: `user_proposed_comparison = []`
+  - set a list user_baseline_comparison to [].  If the end of the rule is reached and this list (and the corresponding proposed list) still have zero length, then the rule passes.  Otherwise the list will include a series of messages for the user about why it didn't pass: `user_baseline_comparison = []`
+  
+  - Check whether the system exists in the user model: `if u_swh_system_and_equip_dict[swh_dist_system_id]:`
+    - check whether the system has SWH uses in the user model: `if u_swh_system_and_equip_dict[swh_dist_system_id]["USES"]:`
+      - get the user swh_use: `u_swh_use = get_component_by_id(U_RMD, swh_use_id)`
+      - if the u_swh_use exists, and is greater than zero: `if u_swh_use != NULL and u_swh_use.use > 0:`
+        - compare the proposed and user models using the function compare_swh_dist_systems_and_components.  This function returns a list of errors.  If the list of errors has a length of 0, then the comparison encountered no issues: `user_proposed_comparison = compare_swh_dist_systems_and_components(P_RMD,U_RMD,user_proposed_str,swh_dist_system_id)`
+        - from here, go directly to the rule assertion: `GO TO RULE ASSERTION`
+
+  -  If the system doesn't exist in the proposed model, we need to return an error: `if swh_dist_system_id not in p_swh_system_and_equip_dict: errors << swh_dist_system_id + " was not found in the Proposed model. 
+ Because there are no SWH loads in the User model, we are expecting the Proposed and Baseline systems to match."`
+  -  If the system doesn't exist in the baseline model, we need to return an error: `if swh_dist_system_id not in b_swh_system_and_equip_dict: errors << swh_dist_system_id + " was not found in the Baseline model. 
+ Because there are no SWH loads in the User model, we are expecting the Proposed and Baseline systems to match."`
+  -  if we got to this line without going to the rule assertion, it means that there is no equivalent system in the user model, or that the user model does not have SWH use.  Compare the proposed and baseline models using the function compare_swh_dist_systems_and_components.  This function returns a list of errors.  If the list of errors has a length of 0, then the comparison encountered no issues: `user_baseline_comparison = compare_swh_dist_systems_and_components(P_RMD,B_RMD,user_proposed_str,swh_dist_system_id)`
+  
+  - now go to rule assertion: `GO TO RULE ASSERTION`
+    
+
+## Rule Assertion: 
+- Case1: if the length of either user_proposed_comparison or user_baseline comparison is 0, then the rule passes: `if (len(user_proposed_comparison) == 0) || (len(user_baseline_comparison) == 0): PASS`
```

Comment:
wouldn't we want the length of both (not either) of these lists to be 0?

Reply:
yes! good catch